tips/TIP-0026/indexer-rest-api.yaml

```diff
@@ -0,0 +1,757 @@
+openapi: 3.0.3
+info:
+  title: IOTA UTXO Indexer REST API
+  description: This document specifies the REST API for IOTA UTXO indexers.
+  contact:
+    email: contact@iota.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  version: 1.0.0
+externalDocs:
+  description: Find out more about IOTA
+  url: 'https://iota.org'
+servers:
+  - url: 'http://127.0.0.1:14265'
+tags:
+  - name: outputs
+    description: Query Extended Outputs.
+  - name: aliases
+    description: Query Alias Outputs.
+  - name: foundries
+    description: Query Foundry Outputs.
+  - name: nfts
+    description: Query NFT Outputs.
+paths:
+  /api/plugins/indexer/v1/outputs:
+    get:
+      tags:
+        - outputs
+      summary: Returns extended outputs filtered based on parameters.
+      description: Returns extended outputs filtered based on parameters.
+      parameters:
```

Comment:
I think the limit parameter is missing
/// Define the page size for the results
Limit(usize),

Reply:
`pageSize` parameter replaces `limit`